docs/modules/reference/pages/billing/subscription/user-subscription.adoc

```diff
@@ -1,50 +0,0 @@
-= Manage Users
```

Comment:
This move broke the link mentioned in "Billing Management.adoc" in How-to-Guides

Reply:
I pushed a big change. I'll redo the navigation tomorrow and you take another look. Stepping out to get some medications.

Reply:
![CleanShot 2024-09-10 at 15 42 17](https://github.com/user-attachments/assets/00f3cc32-fb11-4312-9a9a-fd0a7ca0a6a4)
Not able to see "How To Guides" and "Getting Started" columns